### ISSUE TITLE
Keep numpad up while navigating within mode

### ISSUE THREAD

**Issue by mattave** [RunSignUp-Team/RunSignup-Mobile-Timing-App#40]
Requested by Eric Kramer.

> While in "Finish Line Mode" and recording "Bib Entries"
> My thought process here is, that if you are in Bib Entry mode, you want to stay in bib entry mode (at least we do)
> So, if you tap any button other than "Record" after entering a number, it pops out of Bib Entry mode and the number pad disappears
> The same thing happens while in "Chute Mode", after completing Finish Line Mode

I'm not sure if it would be possible to keep the cursor on the bib entry field so that the numpad never goes down, or if we would have to develop a native numpad that just always stays up in these modes unless you explicitly collapse it down or something like that.

**Comment by nathanael-rsu** [comment]
Android is always finicky with focusing on Text Inputs, so they might be having an issue with that, but it should only potentially be impacting them if they are focused on the Bib Entry Text Input, then they tap a bib number to edit it, and then the close the dialog. On Android, it isn't always re-focusing on the bib entry text field. 

Not sure why else they would be tapping out of Bib Entry mode, though. The whole point is that as long as they are just entering bib numbers and tapping Record, it never closes the keyboard. It sounds like they never want it to exit the keyboard no matter where they tap on the screen, but that's not a great design, and on iPhones, there is literally no other way to dismiss the keyboard.

**Comment by mattave** [comment]
I think that they were navigating whatever mode they were currently in, and didn't like how the num pad goes away when they tap the results rows list or did something else that made the cursor deselect the bib entry field. Maybe it's just what you're describing with Android clearing the focus for the text input field. On iOS you're saying that it doesn't clear the focus when you tap other parts of the interface within a mode?

**Comment by nathanael-rsu** [comment]
I'm saying that there's basically no reason to tap another part of the interface, so I don't understand why they're doing that and not expecting the keyboard to be dismissed. That is standard behavior. As long as they are typing in bibs and tapping record, they should have no issues regardless of OS. If they edit a bib number, they might have to re-select the Text Input on Android.

**Comment by mattave** [comment]
Yeah, I think their complaint is that they are tapping other areas maybe inadvertently or maybe intentionally, and they would like for the bib field to stay focused when doing so. If it's not possible to keep the focus on the bib field we can just tell them that and move on, just wanted to see if there was a possibility.

**Comment by nathanael-rsu** [comment]
I can keep it focused if that's what you want, but I don't think that's standard behavior / I'm not sure how else you would dismiss the keyboard on iOS. I think there should always be a way to dismiss a keyboard.

**Comment by mattave** [comment]
Gotcha. It's interesting that iOS doesn't have a way to manually hide/show a keyboard, but that makes sense to not do this if there's no way to manage the keyboard manually like you can with Android
